s2i/s2i.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: s2i
+spec:
+  parameters:
```

Comment:
given that you have a `default` for `BUILDAH_BUILDER_IMAGE` (your `docker.io/vdemeester/buildah-builder`), how about offering the same here for this S2I builder image with the Dockerfile from the PR? (Or perhaps you wanted to do that only as a follow-up once this is merged' ? Ideally there eventually probably should be a knative-build-templates set up on Docker Hub, and/or Quay.io, which auto builds the containers from this project... Is that anything which there may be any interest for me helping to get that set up?)

Reply:
I'll remove the default `BUILDAH_BUILDER_IMAGE` for now

Reply:
OK sure